<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Matrix Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background-color:rgb(116, 149, 255);
}
QPushButton{
	background-color:white;
	color:rgb(116, 149, 255);
	border-radius:10px;
	font-weight:bold;
}
QPushButton:hover{
	background-color:rgb(116, 149, 255);
	color:white;
	border: 3px solid rgb(43, 92, 255);
}</string>
  </property>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Matricies</string>
   </property>
  </widget>
  <widget class="QPushButton" name="multiplyBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Multiply Matricies</string>
   </property>
  </widget>
  <widget class="QPushButton" name="constBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Multiply Matrix by a constant</string>
   </property>
  </widget>
  <widget class="QPushButton" name="transposeBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Transpose</string>
   </property>
  </widget>
  <widget class="QPushButton" name="determinantBtn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate Determinant</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inverseBtn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Find Inverse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitBtn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:rgb(255, 0, 0);
	color:white;
}
QPushButton:hover{
	background-color:white;
	color:red;
	border: 3px solid red;
}</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 81 20pt &quot;JetBrains Mono NL ExtraBold&quot;;
color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Matrix Calculator</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
